packages/buidler/package.json

```diff
@@ -25,7 +25,6 @@
     "send": "buidler send"
   },
   "dependencies": {
-    "@opengsn/gsn": "git+https://github.com/opengsn/gsn#solc-0.7.5",
     "@openzeppelin/contracts": "3.1.0-solc-0.7",
```

Comment:
This can be removed too

Reply:
No real need as I will delete this directory when finished with hardhat.